src/Package/Impl/DataInspect/Office/CsvAppFileIO.cs

```diff
@@ -69,7 +69,7 @@ internal static class CsvAppFileIO {
             Interlocked.Exchange(ref _busy, 0);
         }
 
-        private static async Task CreateCsvAndStartProcess(IREvaluationResultInfo result, IRSession session, string fileName, IFileSystem fileSystem, IProcessServices processServices) {
+        private static async Task CreateCsvAndStartProcess(IREvaluationResultInfo result, IRSession session, string fileName, IFileSystem fileSystem, IProcessServices processServices, IVsStatusbar statusBar) {
```

Comment:
For Export to CSV I have switched to modal.
For SendToRemote/Source until current command has finished the UI does not allow any other operation.